Dynatrace/dynatrace-operator
# Deprecates prefixed feature flags

# Description

* Deprecates all feature flags with either a "disable-" or "enable-" prefix.
* Provides un-prefixed alternatives which take precedent to the old ones

## How can this be tested?

* Run the unit tests that were added
* Add log lines of the beginning of the operator reconciler to output the feature flag values or whether or not a specific feature is disabled and check if the output aligns with the annotations set in a dynakube

For the highly motivated:
* Test the features that are enabled / disabled by the deprecated feature flags
* Check if the features are still enabled / disabled accordingly with the deprecated flags
* Check if  the features are still enabled / disabled accordingly with the new flags
* Check if  the features are still enabled / disabled accordingly with the new flags taking precedent to the old ones

## Checklist
